terminus-ui/file-upload/src/selected-file.spec.ts

```diff
@@ -63,47 +68,56 @@ const FILE_VIDEO_MOCK = FILE_VIDEO_BLOB as File;
 
 
 describe(`TsSelectedFile`, function() {
-  let newSelectedFile: TsSelectedFile;
   const createFile = (
     file = FILE_MOCK,
     constraints: TsFileImageDimensionConstraints | undefined = CONSTRAINTS_MOCK,
     types: TsFileAcceptedMimeTypes[] = ['image/png', 'image/jpg'],
     size = (10 * 1024),
-    ratio = [{ widthRatio: 1, heightRatio: 1 }],
-  ): TsSelectedFile => {
-    return newSelectedFile = new TsSelectedFile(
-      file,
-      constraints ? constraints : undefined,
-      types,
-      size,
-      ratio,
-    );
-  };
+    ratio = [{
+      widthRatio: 1,
+      heightRatio: 1,
+    }],
+  ): TsSelectedFile => new TsSelectedFile(
+    file,
+    constraints ? constraints : undefined,
+    types,
+    size,
+    ratio,
+  );
 
 
   // Mock `FileReader` and `Image`:
   beforeEach(() => {
     // Mock FileReader
     class DummyFileReader {
-      onload = jest.fn();
-      addEventListener = jest.fn();
-      readAsDataURL = jest.fn().mockImplementation(function(this: FileReader) { this.onload({} as Event); });
-      // tslint:disable: max-line-length
-      result = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEgAAABIAQMAAABvIyEEAAAAA1BMVEXXbFn0Q9OUAAAADklEQVR4AWMYRmAUjAIAAtAAAaW+yXMAAAAASUVORK5CYII=';
-      // tslint:enable: max-line-length
+      public onload = jest.fn();
+      public addEventListener = jest.fn();
+      public readAsDataURL = jest.fn().mockImplementation(function(this: FileReader) {
+        this.onload({} as Event);
+      });
+      // eslint-disable-next-line max-len
+      public result = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEgAAABIAQMAAABvIyEEAAAAA1BMVEXXbFn0Q9OUAAAADklEQVR4AWMYRmAUjAIAAtAAAaW+yXMAAAAASUVORK5CYII=';
     }
     // Not sure why any is needed
```

Comment:
do we need a `// tslint:disable-next-line no-any`

Reply:
Spec files allow any by default now.